src/libraries/System.Net.Http.Json/src/System/Net/Http/Json/JsonContent.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.IO;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.Net.Http.Json
+{
+    public sealed partial class JsonContent : HttpContent
+    {
+        internal const string JsonMediaType = "application/json";
+        internal const string JsonType = "application";
+        internal const string JsonSubtype = "json";
+        private static MediaTypeHeaderValue DefaultMediaType
+            => MediaTypeHeaderValue.Parse(string.Format("{0}; charset={1}", JsonMediaType, Encoding.UTF8.WebName));
```

Comment:
```suggestion
            => new MediaTypeHeaderValue(JsonMediaType) { CharSet = "utf-8" };
```

```C#
using BenchmarkDotNet.Attributes;
using BenchmarkDotNet.Running;
using System.Net.Http.Headers;
using System.Text;

[MemoryDiagnoser]
public class Program
{
    static void Main(string[] args) => BenchmarkSwitcher.FromAssemblies(new[] { typeof(Program).Assembly }).Run(args);

    internal const string JsonMediaType = "application/json";

    [Benchmark]
    public MediaTypeHeaderValue V1() => MediaTypeHeaderValue.Parse(string.Format("{0}; charset={1}", JsonMediaType, Encoding.UTF8.WebName));

    [Benchmark]
    public MediaTypeHeaderValue V2() => MediaTypeHeaderValue.Parse($"{JsonMediaType}; charset={Encoding.UTF8.WebName}");

    [Benchmark]
    public MediaTypeHeaderValue V3() => MediaTypeHeaderValue.Parse("application/json; charset=utf-8");

    [Benchmark]
    public MediaTypeHeaderValue V4() => new MediaTypeHeaderValue(JsonMediaType) { CharSet = "utf-8" };
}
```

| Method |     Mean |   Error |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|------- |---------:|--------:|--------:|-------:|------:|------:|----------:|
|     V1 | 307.2 ns | 0.78 ns | 0.73 ns | 0.0634 |     - |     - |     400 B |
|     V2 | 220.1 ns | 0.37 ns | 0.33 ns | 0.0637 |     - |     - |     400 B |
|     V3 | 199.1 ns | 0.72 ns | 0.67 ns | 0.0496 |     - |     - |     312 B |
|     V4 | 124.7 ns | 2.11 ns | 1.97 ns | 0.0293 |     - |     - |     184 B |

Reply:
Shouldn't we prefer the strong type `Encoding.UTF8.WebName`  instead of `utf-8` string?

Reply:
What do you mean?  WebName returns a string.

Reply:
I mean, your suggestions sets `CharSet = "utf-8"`; shouldn't that say `CharSet = Encoding.UTF8.WebName`?

Reply:
Oh I see, it is ~20ns for the virtual call to `Encoding.UTF8.CharSet`. Shold we add another constant called `JsonEncoding`?.

Reply:
Even perf aside, there's little reason to go to `Encoding.UTF8.CharSet` to get the string we know needs to be `"utf-8"`.  From my perspective, it doesn't need to be put into a named const, but I don't have a strong preference.